### PR TITLE
example: Disable controllers in NonVR

### PR DESCRIPTION
**Pull request by timothyhale**

### CODE REVIEW

File: example/SpatialAudioLab.wlp [hunk 1]
```diff
@@ -17,6 +17,18 @@
                 0.9999998,
                 1.0,
                 0.9999998
+            ],
+            "components": [
+                {
+                    "type": "wasd-controls"
```
Comment (Squareys) [inline example/SpatialAudioLab.wlp:23]: You should set the head to NonVrCamera, otherwise the mouse-look does not affect the wasd direction

Comment (Squareys) [reply example/SpatialAudioLab.wlp:23]: Also, I think the NonVrCamera has a teleport component attached that you might want to remove?

File: example/js/button.js [hunk 2]
```diff
@@ -54,11 +54,13 @@ export class ButtonComponent extends Component {
             audioFile: 'sfx/click.wav',
             HRTF: true,
             isStationary: true,
+            maxVolume: 0.5
         });
         this.unclick = this.object.addComponent(AudioSource, {
             audioFile: 'sfx/unclick.wav',
             HRTF: true,
```
Comment (Squareys) [inline example/js/button.js:61]: This property should probably be `hrtf` according to naming convention.

File: example/SpatialAudioLab.wlp (hunk 1 above)
Comment (timothyhale) [reply example/SpatialAudioLab.wlp]: it didnt have a teleport

**Pull request merged**